react-1/src/ValueChooser.css

```diff
@@ -0,0 +1,10 @@
+.ValueChooser-button {
+  background: #ccc;
+  border: 1px solid #aaa;
+}
+
+.ValueChooser-button_selected {
```

Comment:
Out of curiosity: is this a real life naming convention?

Reply:
This is freely inspired from BEM :)

Reply:
=> https://css-tricks.com/bem-101/
(j'ai inversé `-` et `_`, et eux ils doublent pour pouvoir avoir des noms avec des `-` et des `_` :) -- mais bon, c'est le principe)